src/remux/passthrough-remuxer.ts

```diff
@@ -256,6 +256,12 @@ function getParsedTrackCodec(
   if (parsedCodec === 'avc1' || type === ElementaryStreamTypes.VIDEO) {
     return 'avc1.42e01e';
   }
+  if (parsedCodec === 'fLaC') {
+    return 'fLaC';
+  }
+  if (parsedCodec === 'Opus') {
+    return 'Opus';
```

Comment:
If the return values match `parsedCodec`, then just return `parsedCodec`:
```suggestion
  if (parsedCodec === 'fLaC' || parsedCodec === 'Opus') {
    return parsedCodec;
```